### ISSUE TITLE
Add VpcId to Outputs list

### ISSUE THREAD

**Issue by cirocosta** [docker/for-aws#36]
### Expected behavior

Have the `VPCID` exposed in the Outputs lists

### Actual behavior

Exposes only: 
- defaultdnstarget 
- zoneavailabilityComment
- managers
- elbdnszoneid 

### Information

I'd like to have the VPC-ID exposed in the outputs so that I can easily grab it from a tool like terraform and create a private hosted zone with such ID so that the instances can resolve internal dns queries. 

To have the functionality I just updated the CF template:

```json
    "Outputs": {
        "VPCID": {
            "Description": "Use this as the VPC for configuring Private Hosted Zones",
            "Value": {
                "Ref": "Vpc"
            }
        },
        "DefaultDNSTarget": {
...
```

so that I have:

<img width="875" alt="screen shot 2017-04-28 at 11 41 05" src="https://cloud.githubusercontent.com/assets/3574444/25533622/98efa8d0-2c07-11e7-9d47-d42840b4cec9.png">

which can then be used in terraform:

```hcl
resource "aws_route53_zone" "private-domain" {
  name   = "example-domain"
  vpc_id = "${aws_cloudformation_stack.docker-for-aws.outputs["VPCID"]}"
}

resource "aws_route53_record" "private-domain" {
  zone_id = "${aws_route53_zone.private-domain.zone_id}"
  name    = "self.example-domain"
  type    = "A"
  ttl     = "300"

  records = [
    "127.0.0.1",
  ]
}
```

Maybe someone else also has a use case for that.

### Steps to reproduce the behavior

  1. Deploy the latest edge CF template (17.04.0-edge CE)
  2. check the outputs

**Comment by kencochrane** [comment]
@cirocosta thanks for the suggestion, seems reasonable, I'll see what I can do to add it to the next release.

**Comment by kencochrane** [comment]
This is available in 17.05-ce edge. closing
